addons/advanced_fatigue/stringtable.xml

```diff
@@ -116,13 +116,15 @@
             <Chinesesimp>抖动因数</Chinesesimp>
             <Chinese>抖動因素</Chinese>
             <French>Facteur de stabilisation</French>
+            <Italian>Fattore di oscillazione</Italian>
```

Comment:
>French translation

:thinking:

Reply:
i'll check but it's likely another PR deleted that line before I PR'd this.

Reply:
it's due to how the merge was done @dedmen check the merge commit.